pennylane/devices/default_qubit.py

```diff
@@ -373,7 +376,7 @@ def preprocess(
                 parameters needed to fully describe the execution.
 
         Returns:
-            TransformProgram, ExecutionConfig: A transform program that when called returns QuantumTapes that the device
+            ~.TransformProgram, ~.ExecutionConfig: A transform program that when called returns QuantumTapes that the device
```

Comment:
didn't fix it. Romain was dealing with links for TransformProgram on his PR, not sure how to make it work. the config was already working on master

Reply:
I'll remove these changes in that case :)